website/content/docs/commands/web.mdx

```diff
@@ -0,0 +1,84 @@
+
+### read
+Like the CLI [`read`](/vault/docs/commands/read) command, the Web REPL `read` performs an HTTP GET on a given path.
+
+Also like the CLI `read`, the REPL implements `-field` and `-format` flags. The output defaults to "table" format, but also supports "json" output.
+
+### write
+This is a Web REPL implementation of the CLI [`write`](/vault/docs/commands/write) command - it will perform an HTTP POST to the given path with the given data. Notably the special "**@**" syntax from the CLI is not implemented - data must be specified as arguments to the command. Because of this, any API fields that require data structures not expressible as arguments are not supported.
+This is the major shortcoming of the Vault Web REPL.
+
+
+## Command options
+
+### field
+If you are only interested in a single field in the response, you can use the `-field` flag which will specify that the REPL should only print that field in the response. Using this in conjucntion with `-wrap-ttl` is nice because you can return just the wrapped token by doing something like this:
+```
+kv-get secret/one -wrap-ttl=10m -field=token
+```
+### force
+Some delete paths require using `-force` as a confirmation that the delete is intentional. Using `delete` without this flag on a path that needs it will result in an error that tells you `-force` is required.
+
+### format
+`read` commands default to the "table" format, if you would like to see the JSON format of a response, you can pass `-format=json`. This is most often useful for responses that have deeply nested objects that don't fit well in the table format.
+
+### metadata
+As mentioned above in the `kv-get` section - this flag is shorthand for `-field=metadata` which is useful when reading secrets from the KVv2 secrets engine.
+
+### wrap-ttl
+Like the `vault` CLI, the Web REPL supports creating [response-wrapping tokens](/vault/docs/concepts/response-wrapping#response-wrapping-token-creation).The format is also the same: it can be an integer that sets the wrapping TTL for a number of seconds, or it can be a string that specifys the length of the duration in seconds (`15s`), miniutes (`20m`), or hours (`25h`).
+
+## REPL-specific commands
+
+### api
+This will navigate you to an interactive OpenAPI explorer in the Vault UI. This explorer will only contain the paths your current `VAULT_TOKEN` has permissions to operate on.
+
+### clear
+This will clear all of the output currently in the Web REPL log.
+
+### clearall
+This clears the output _and_ the command history in the Web REPL. After running `clearall`,  arrowing up won't show previsouly executed commands.
```

Comment:
```suggestion
This clears the output _and_ the command history in the Web REPL. After running `clearall`, arrowing up won't show previsouly executed commands.
```

Reply:
If only I had spell check in my editor 🤦